nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -271,6 +271,12 @@ php.override {
 </programlisting>
     </para>
    </listitem>
+   <listitem>
+    <para>
+      Option <literal>services.nginx.stateDir</literal> removed. Nginx Logs directory moved to <literal>/var/log/nginx</literal>
+      and nginx cache dir moved to <literal>/var/cache/nginx</literal>.
```

Comment:
The Nginx log directory has been moved to <literal>/var/log/nginx</literal>, the cache directory to <literal>/var/cache/nginx</literal>. The option <literal>services.nginx.stateDir</literal> has been removed.

Reply:
Thanks, fixed.